src/components/Common/CrudlDocument.vue

```diff
@@ -43,7 +43,12 @@
             Toggle all
           </button>
 
-          <button class="btn btn-small waves-effect waves-light margin-right-5 primary" @click.prevent="create"><i class="fa fa-plus-circle left"></i>Create</button>
+          <button class="btn btn-small waves-effect waves-light margin-right-5 primary"
+                  @click.prevent="create",
+                  v-if="displayCreate">
```

Comment:
We have not to hide element, but disable it with a help text.

See exemple here: [src/components/Data/Indexes/List.vue#L19](https://github.com/kuzzleio/kuzzle-backoffice/blob/master/src/components/Data/Indexes/List.vue#L19)